Formula/lftp.rb

```diff
@@ -1,23 +1,33 @@
 class Lftp < Formula
   desc "Sophisticated file transfer program"
   homepage "https://lftp.yar.ru/"
-  url "https://lftp.yar.ru/ftp/lftp-4.6.5.tar.xz"
-  sha256 "1fd0920a1791ce0e9e39ffce77ae6619e5dc665f16e9380bafbfc69411eeb71e"
+  url "https://lftp.yar.ru/ftp/lftp-4.7.2.tar.xz"
+  sha256 "475d7699b1390f951efee867ba1ad600f78329e13fd2a04d92f82bfffb70d872"
 
   bottle do
     sha256 "f341d62601963f968e46907f88e1e96d21154deb2b60d97bd3bac27a37816575" => :el_capitan
     sha256 "182979ca79b7ea9d9e70e9fefab990d5d238da709ce0e17edf5ec7166b976fb1" => :yosemite
     sha256 "dc44f63a8f1767af2b4ed2008c199377eabd117339f01ee17bb73aa844cf3dee" => :mavericks
   end
 
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
   depends_on "pkg-config" => :build
   depends_on "readline"
-  depends_on "openssl"
+  depends_on "openssl" if MacOS.version < :yosemite
+
+  patch do
```

Comment:
Please note that link in the formula with the standard `#`. We can't realistically expect future contributors to comb through old PRs for answers.